src/components/HomeLeftContainer.vue

```diff
@@ -1,19 +1,45 @@
 <script setup lang="ts">
 import LevelDialog from "../components/LevelDialog.vue";
-import { ref } from "vue";
+import { ref, reactive, computed } from "vue";
 import { storeToRefs } from "pinia";
 import { userStore } from "../stores/user";
 let showLevelDialog = ref<boolean>(false);
 const langMenu: string[] = ["JavaScript", "Java"];
 const levelMenu: string[] = ["初級", "中級", "上級"];
+const user = userStore();
+const { lang, checkLang, level, checkLevel } = storeToRefs(user);
+let langStatus = ref<boolean>(false);
+let levelStatus = ref<boolean>(false);
+const langInput = computed({
+  get: () => langStatus.value,
+  set: (value: boolean) => (langStatus.value = value),
+});
+const levelInput = computed({
+  get: () => levelStatus.value,
+  set: (value: boolean) => (levelStatus.value = value),
+});
```

Comment:
読みづらいので、関数と関数の間は1行空けてください
なんでも詰めたら良いというものではありません。
```suggestion
const langInput = computed({
  get: () => langStatus.value,
  set: (value: boolean) => (langStatus.value = value),
});
// 空行
const levelInput = computed({
  get: () => levelStatus.value,
  set: (value: boolean) => (levelStatus.value = value),
});
```